Makefile

```diff
@@ -0,0 +1,34 @@
+# Get protoc commands
+PROTOC := $(shell command -v protoc)
+PROTOC_GEN_GO := $(shell command -v protoc-gen-go)
+PROTOC_GEN_GO_GRPC := $(shell command -v protoc-gen-go-grpc)
+
+# Get package manager
+PACKAGE_MGR := $(shell ([ -f /etc/redhat-release ] && echo yum) || ([ -f /etc/debian_version ] && echo apt-get))
+
+all: checkenv build
+
+build: checkenv
+	protoc --go_out=lib/go/ *.proto
+	#protoc --go_out=lib/go --go_opt=paths=source_relative *.proto
+	#protoc --go-grpc_out=lib/go --go-grpc_opt=paths=source_relative *.proto
+
+checkenv:
+	mkdir -p google/protobuf
```

Comment:
this needs to be installed in some /tmp/xyz folder?

Reply:
Downloading in our own repo